apps/web/src/components/purchase-form/crypto/sections/crypto-form-wc.tsx

```diff
@@ -93,27 +81,15 @@ export default function CryptoFormWalletConnect({
       return
     }
 
-    // Check USDC balance
-    const usdcBalance = formatToDecimal(usdcAsset.amount, usdcAsset.decimals)
-    const currencyBalance = formatToDecimal(
-      usdcAsset.amount,
-      usdcAsset.decimals,
-      currency,
-      conversionRate
-    )
-
-    const usdcBalanceInt = formatFloatToInt(usdcBalance)
-
-    const usdcPriceInt = formatFloatToInt(price)
-    const currencyPriceInt = formatFloatToInt(price, currency, conversionRate)
```

Comment:
This was kinda weird that we're converting an int to a float and back to an int again. It'd make sense if we were dealing with varying currencies/coins, but here we're only interacting with USDC which appears to be returned in cents. Lucky us.